src/wot.py

```diff
@@ -0,0 +1,34 @@
+import os
+from tabulate import tabulate
+from collections import OrderedDict
+
+from network_tools import *
+from tools import *
+from constants import *
+
+
+def received_sent_certifications(ep, id):
+    """
+    check id exist
+    many identities could exist
+    retrieve the one searched
+    get id of received and sent certifications
+    display on a chart the result with the numbers
+    """
+    if get_pubkeys_from_id(ep, id) == NO_MATCHING_ID:
+        print(NO_MATCHING_ID)
+        sys.exit(1)
+    certs = request(ep, "wot/lookup/" + id)["results"]
+    for cert in certs:
+        if cert["uids"][0]["uid"].lower() == id.lower():
+           certs = cert
```

Comment:
Agreed with that `break` is missing.
There shouldn't be two identities but why not. It's proper that way.
The id must be found as [it's checked before](https://github.com/duniter/silkaj/pull/50/files#diff-403c9cae1b2f2f9c04f8952279d8c6a9R18)